Cargo.toml

```diff
@@ -46,7 +46,7 @@ aws-lc-rs = { version = "1.12", default-features = false }
 base64 = "0.22"
 bencher = "0.1.5"
 brotli = { version = "8", default-features = false, features = ["std"] }
-brotli-decompressor = "4.0.1" # 4.0.1 required for panic fix
+brotli-decompressor = "5.0.0" # 4.0.1 required for panic fix
```

Comment:
I think we can drop this comment now (either as a tacked-on-commit to this branch, or in a follow-up PR).